projects/angular/src/wizard/wizard-stepnav-item.ts

```diff
@@ -158,4 +176,22 @@ export class ClrWizardStepnavItem {
       throw new Error('Wizard stepnav item is not associated with a wizard page.');
     }
   }
+
+  private ensureCurrentStepIsScrolledIntoView() {
+    // Don't use "smooth" scrolling when the wizard is first opened to avoid a delay in scrolling the current step into view.
+    // The current step when the wizard is opened might not be the first step. For example, the wizard can be closed and re-opened.
+    let scrollBehavior: ScrollBehavior = 'auto';
+
+    return this.navService.currentPageChanged.pipe(
+      startWith(this.navService.currentPage),
+      tap(currentPage => {
```

Comment:
Could we use skip instead of the variable "skipNextScroll" to skip the first emitted change? Or we need it specifically after the click?
```suggestion
      skip(1), 
      startWith(this.navService.currentPage),
      tap(currentPage => {
```